tracer/src/Datadog.Trace/SpanContext.cs

```diff
@@ -23,6 +23,13 @@ public class SpanContext : ISpanContext, IReadOnlyDictionary<string, string>
             HttpHeaderNames.DatadogTags,
         };
 
+        /// <summary>
+        /// An <see cref="ISpanContext"/> with default values. Can be used as the value for
+        /// <see cref="SpanCreationSettings.Parent"/> in <see cref="Tracer.StartActive(string, SpanCreationSettings)"/>
+        /// to specify that the new span should not inherit the currently active scope as its parent.
+        /// </summary>
+        public static readonly ISpanContext Empty = new ReadOnlySpanContext(traceId: 0, spanId: 0, serviceName: null);
```

Comment:
Do we really want a new type, of can we just use `SpanContext`? The `ISpanContext` interface is read-only, but there's nothing stopping users from casting this to `SpanContext` and modifying it.

Reply:
Thanks, I will leave it then.